requirements.txt

```diff
@@ -0,0 +1,6 @@
+Flask==0.10.1
+itsdangerous==0.24
```

Comment:
itsdangerous, Jinja2, MarkupSafe and Werkzeug are dependencies of Flask.
wheel is installed by virtualenv when you create one.